src/cli/input-from-terminal.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
```

Comment:
Removed this function from format file.
For some reason, I was not able to mock this function even if I exported it from `format` file.
Had to extract it to a separate file in order to mock this.

Reply:
This can't be mocked in production, we need add it to `src/common/third-party.js`(you can rename it to `modules-to-mock.js`) and use it from there.